### PR TITLE
Update README.md for community link

### PR DESCRIPTION
**Pull request by moshfiqrony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ This repo contains the source code and documentation powering [bn.react.dev](htt
 
 ## Getting started
 
+### Join the Bengali discord community
+Hi dev, We are Bengali React doc translation community welcome you to join the discord server - https://github.com/reactjs/bn.react.dev/issues/26
```
Comment (nafistiham) [inline README.md:8]: should have been the server link?

**Review by nafistiham — CHANGES_REQUESTED** [review]
Vai, I think you wanted to put the server link maybe?

File: README.md (hunk 1 above)
Comment (moshfiqrony) [reply README.md:8]: Bhai I try to keep the issue as the main point for community link and update. That's why I added the issue intentionally. Like for progress we follow one issues.

Comment (nafistiham) [reply README.md:8]: Oh! now I got it. I guess in that case we could make the code inline and a bit paraphrase like this?

Hi dev, we are Bengali React doc translation community. We welcome you to [join our discord server.](https://github.com/reactjs/bn.react.dev/issues/26)

Comment (nafistiham) [reply README.md:8]: Also, I'd suggest making the line in the issue inline as well. Like...
Anyway, approving the PR.

Here is the discord server invitation [link.]( https://discord.gg/6YG28yBF)

**Review by nafistiham — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (nafistiham) [reply README.md:8]: Ahh! I thought approving wouldn't just merge it immediately. I am so sorry.

**Pull request merged**